src/components/Button/Button.js

```diff
@@ -21,11 +22,12 @@ export const Button = (props) => {
   return (
     <>
       <button
-        className={`rx-button-${type} rx-button-${corners} rx-button-icon-position-${iconPosition} ${className}`}
+        className={`rx-button-${style} rx-button-${corners} rx-button-icon-position-${iconPosition} ${className}`}
```

Comment:
as mentioned in chat, i think we should use display instead of style, as style is used for custom inline styling